apps/web/src/components/layout/components/v2/SyncInfoModalTrigger.tsx

```diff
@@ -15,11 +52,65 @@ export function SyncInfoModalTrigger() {
 
   return (
     <>
-      <Button size="xs" Icon={IconOutlineCloudUpload} onClick={toggleSyncInfoModalShow}>
-        Sync
-      </Button>
+      <Tooltip
+        disabled={isSynced}
+        width="auto"
+        label={<Text>You have un-synced changes in your local application.</Text>}
+        withinPortal
+      >
+        <Container>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Do we need this Container? Ideally we should use the Panda JSX elements to handle basic layouts and wrappers (like `Box` or `Flex` or `Stack`)‏

Reply:
Let me know what will be the best solution for this case, i hope you don't mind i need to merge this one as we want to deploy this change in the current version.